str.h

```diff
@@ -65,7 +65,7 @@ namespace minsky
     for (size_t i=0; i<s.length(); ++i)
       switch (s[i])        
         {
-        case '\\': case '{': case '}':
+        case '\\': //case '{': case '}':
```

Comment:
So braces are now included in valueIds, but not backslashes. Did you identify there were problems with backslashes?

Also spaces are still confused with underscores. So "a_b" and "a b" map to the same variable.

What did you think about using latexToPango as the conversion string? The only time a backslash would appear in the output is if the user types "\backslash". And there is probably a unicode replacement we could use.

Reply:
> So braces are now included in valueIds, but not backslashes. Did you identify there were problems with backslashes?

Yes, the test compareSimulationWithLogged.sh fails because of missing backslashes.

> What did you think about using latexToPango as the conversion string? The only time a backslash would appear in the output is if the user types "\backslash". And there is probably a unicode replacement we could use.

Yes, I replaced stripActive in VariableValue.h, VariableValue.cc and equations.cc by latexToPango, but the assert on line  625 of equations.cc: assert(minsky.variableValues.count(valueId)); fails. I commented it out to see what would happen, but Minsky crashes. I will debug it today. 

When I couldn't make progress on adding the Curl Requests C++ wrapper as a submodule yesterday (damn cmake!), I switched back to this bug, but then after I got stuck, I switched to ticket 1163 (and 1166 which I think is related to it), but I still haven't identified the cause. I will GDB both today.